lib/fathead/mdn_css/parse.pl

```diff
@@ -236,27 +236,25 @@ sub make_redirect {
     output entry with redirect field
 =cut
 
-    my ($title) = @_;
+    my ( $title, $title_clean ) = @_;
     my @data;
     my $outputline;
     if ( $title =~ m/\(\)$/ ) {
         say "\nFound Function: $title";
         my $temp = $title;
         $temp =~ s/\(\)$/ function/;
         $outputline = join "\t",
-          ( $temp, 'R', $title, '', '', '', '', '', '', '', '', '', '' );
+          ( $temp, 'R', $title_clean, '', '', '', '', '', '', '', '', '', '' );
         push @data, $outputline;
         $temp = $title;
         $temp =~ s/\(\)$//;
         $outputline = join "\t",
-          ( $title, 'R', $temp, '', '', '', '', '', '', '', '', '', '' );
```

Comment:
this should be flipped so `$temp` (e.g. `dir`) redirects to `$title` (e.g. `dir()`)